funnel/util.py

```diff
@@ -40,8 +43,32 @@ def format_twitter_handle(handle):
     parsed_handle = urlparse(handle)
     if (
             (parsed_handle.netloc and parsed_handle.netloc != 'twitter.com') or
-            (not parsed_handle.netloc and len(handle) > 16)
+            (not parsed_handle.netloc and len(handle) > 16) or
+            (not parsed_handle.path)
     ):
         return None
 
     return unicode([part for part in parsed_handle.path.split('/') if part][0]).replace('@', '')
+
+
+def split_name(fullname):
+    """
+    Splits a given fullname into two parts
+    a first name, and a concanetated last name.
+    Eg: "ABC DEF EFG" -> ("ABC", "DEF EFG")
+    """
+    name_splits = fullname.split()
+    return name_splits[0], " ".join([s for s in name_splits[1:]])
```

Comment:
Added check for null case. But, if a name doesn't have a space, this just returns an empty string for last name.